.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: ci
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.ref || github.sha }}
+  cancel-in-progress: ${{ github.event_name == 'pull_request' }}
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
```

Comment:
Should we use our runners?

Reply:
This is a public repo, I think it's fine for the load is going to have